docs/source/openshift/openshift-client-binary.adoc

```diff
@@ -27,26 +28,26 @@ export PATH="/Users/john/.minishift/cache/oc/v1.5.0:$PATH"
 ----
 
 [[minishift-oc-context]]
-== Minishift oc context
+== Minishift oc Context
```

Comment:
We're supposed to use title caps in all headings, according to the openshift guidelines https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#topic-titles-and-section-headings

In fact, I'd like to suggest that we ditch the term "oc context", now that I dug a little into it and you also confirmed that it's a CLI Profile, and I would propose that we rename it as such so that we're aligned with the terms. I don't know where the term "oc context" came from" but it's not accurate, even though the command uses that term (which IMO should have been aligned in the first place but I guess it's too late now to change that).